17:20:48.665 [main] ERROR  - Error
17:20:48.668 [main] ERROR  - err
java.lang.NullPointerException
	at Main.main(Main.java:20) [pharmacy/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_91]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_91]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_91]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_91]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144) [idea_rt.jar:?]
17:21:10.840 [main] ERROR  - Error
17:21:10.842 [main] ERROR  - java.lang.NullPointerException
18:52:40.513 [main] ERROR by.training.pharmacy.dao.impl.database.DatabasePrescriptionDAO - Method: DatabasePrescriptionDAO.getUsersPrescriptions
by.training.pharmacy.dao.exception.DaoException: Can not load users prescriptions with login = 'vladikxd4' from database
	at by.training.pharmacy.dao.impl.database.DatabasePrescriptionDAO.getUsersPrescriptions(DatabasePrescriptionDAO.java:46) [pharmacy/:?]
	at Main.main(Main.java:13) [pharmacy/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_91]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_91]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_91]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_91]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144) [idea_rt.jar:?]
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'us_login' in field list is ambiguous
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_91]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_91]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_91]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_91]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.Util.getInstance(Util.java:387) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:934) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3966) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3902) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2526) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2673) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2549) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1861) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1962) ~[mysql-connector-java-5.1.39-bin.jar:5.1.39]
	at by.training.pharmacy.dao.impl.database.DatabaseDAO.readFromDatabase(DatabaseDAO.java:37) ~[pharmacy/:?]
	at by.training.pharmacy.dao.impl.database.DatabasePrescriptionDAO.getUsersPrescriptions(DatabasePrescriptionDAO.java:44) ~[pharmacy/:?]
	... 6 more
